src/components/alert/alertCenter.svelte

```diff
@@ -91,7 +91,7 @@
         <div slot="actions">
           {#each alert.actions as action}
             <Button
-              kind={action.kind ?? 'filled'}
+              kind={action.kind || 'filled'}
```

Comment:
cc @fallaciousreasoning this seems needed to use in projects that don't know about `??` (like brave-core's webpack v4!) as I don't think typescript sees these svelte parts, only the svelte compiler

Reply:
:facepalm: